### PR TITLE
minor corrections

### PR DESCRIPTION
**Pull request by vibhorgupta-gh**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request

- Replace this with your change description

### CODE REVIEW

**Review by akshatnitd — CHANGES_REQUESTED** [review]
Correct the typo of Linkedin

**Review by abishekvashok — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ To add a mentor open the file `mentors.yml` in `_data` folder and please maintai
   twitter: Mentor's Twitter id
   facebook: Mentor's Facebook id
   Linkedin: Mentor's Linkein Page
```
Comment (ba11b0y) [inline README.md]: Please correct the typo

**Review by ba11b0y — CHANGES_REQUESTED** [review]

**Review by anshumanv — CHANGES_REQUESTED** [review]
Conflicts

**Comment by ParthS007** [comment]
@VibhorCodecianGupta  Please resolve conflicts and your PR is ready to merge !

**Review by isuruAb — COMMENTED** [review]
@VibhorCodecianGupta  please resolve conflicts

**Comment by vibhorgupta-gh** [comment]
Hello! Apologies for the extremely late reply. I have resolved the existing conflicts, and I think that this must be it. Please review :D

**Review by ParthS007 — APPROVED** [review]

**Pull request merged**